### PR TITLE
line space

### PR DESCRIPTION
**Pull request by sareh88**

### CODE REVIEW

File: lib/models/user.model.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+
 const contact = new Schema({
 	fullName: { type:String, required:true },
 	avatarURL: { type:String, required:true },
 	userId: { type:Schema.Types.ObjectId, required:true, ref: 'User'}
 })
+
 const userSchema = new Schema({
 	emailAddress: { type: String , lowercase: true, unique : true},
     password: String, // Do not forget, this is an SHA-512 Digest, NOT pure password
```
Comment (aemal) [inline lib/models/user.model.js]: indentation please!! 🗡

Comment (sareh88) [reply lib/models/user.model.js]: done

File: lib/models/user.model.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const contact = new Schema({
 	avatarURL: { type:String, required:true },
 	userId: { type:Schema.Types.ObjectId, required:true, ref: 'User'}
 })
-
+//1
```
Comment (aemal) [inline lib/models/user.model.js]: Change this to //2

Comment (sareh88) [reply lib/models/user.model.js]: done

**Pull request merged**